From 0fb351dfe57c14586085f10c76d17ed4437bba1a Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Fri, 18 Jun 2010 16:11:40 -0300
Subject: [PATCH 5/8] smbios: set system manufacturer/product-name to Red Hat/KVM

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1276877502-7483-6-git-send-email-ehabkost@redhat.com>
Patchwork-id: 10000
O-Subject: [RHEL6 seabios PATCH 5/7] smbios: set system
	manufacturer/product-name to Red Hat/KVM
Bugzilla: 593317
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Gleb Natapov <gleb@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

Bugzilla: 593317
Upstream status: not applicable

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/config.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/config.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/config.h b/src/config.h
index e9ea230..97a4289 100644
--- a/src/config.h
+++ b/src/config.h
@@ -15,8 +15,8 @@
 /* SMBIOS fields */
 #define CONFIG_SMBIOS_VENDOR "Seabios"
 #define CONFIG_SMBIOS_VERSION "0.5.1"
-#define CONFIG_SMBIOS_MANUFACTURER CONFIG_APPNAME
-#define CONFIG_SMBIOS_PROD_NAME CONFIG_APPNAME
+#define CONFIG_SMBIOS_MANUFACTURER "Red Hat"
+#define CONFIG_SMBIOS_PROD_NAME "KVM"
 #define CONFIG_SMBIOS_TYPE3_NAME CONFIG_APPNAME
 #define CONFIG_SMBIOS_CPU_NAME CONFIG_APPNAME
 
-- 
1.7.3.2

